config/index.js

```diff
@@ -82,8 +85,10 @@ const config = {
 const env = process.env.NODE_ENV || 'development';
 const knexConfig = require(path.join(__dirname, '..', './knexfile'))[env];
 const knex = require('knex')(knexConfig);
-config.bookshelf = require('bookshelf')(knex);
-config.bookshelf.plugin('registry');
-config.bookshelf.plugin('virtuals');
+const bookshelf = require('bookshelf')(knex);
+
+bookshelf.plugin('registry');
+bookshelf.plugin('virtuals');
+config.bookshelf = bookshelf;
```

Comment:
Same here, hence the agreement. But it's nice to have one way to do it that is consistent throughout the code base.